test/computePrizeDistributionFromTicketAverageTotalSupplies.test.ts

```diff
@@ -0,0 +1,50 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { computePrizeDistributionFromTicketAverageTotalSupplies } from "../src";
+import { PrizeTier } from "../src/types";
+const debug = require("debug")("v4-js-core:test");
+
+describe("computePrizeDistributionFromTicketAverageTotalSupplies", () => {
+  it("should succeed to calculate a valid PrizeDistribution", async () => {
+    const draw = {
+      winningRandomNumber: BigNumber.from(
+        "21288413488180966377126236036201345909019919575750940621513526137694302720820"
+      ),
+      drawId: 1,
+      timestamp: 1634410924,
+      beaconPeriodStartedAt: 1634324400,
+      beaconPeriodSeconds: 86400,
+    };
+
+    const prizeTier: PrizeTier = {
+      bitRangeSize: 2,
+      endTimestampOffset: 0,
+      maxPicksPerUser: 2,
+      expiryDuration: 86400,
+      drawId: 1,
+      prize: BigNumber.from('100000000000000'),
+      tiers: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,]
+    }
+
+    const expectation = {
+      bitRangeSize: 2,
+      matchCardinality: 2,
+      tiers: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+      maxPicksPerUser: 2,
+      expiryDuration: 86400,
+      numberOfPicks: BigNumber.from({ _hex: "0x03", _isBigNumber: true }),
```

Comment:
Not very legible. Why not use a number instead of a BigNumber in `BigNumber.from` ?